src/accordion/stories/Accordion.tsx

```diff
@@ -0,0 +1,200 @@
+import * as React from "react";
+
+import "./index.css";
+import {
+  useAccordionState,
+  Accordion,
+  AccordionTrigger,
+  AccordionPanel,
+  AccordionState,
+} from "../index";
+
+export type AccordionComponentInitialState = {
```

Comment:
We could use "AccordionInitialState" from the AccordionState.ts right?

Reply:
Agreed, we can use it.